sdk/storage/storage-blob-changefeed/review/storage-blob-changefeed.api.md

```diff
@@ -0,0 +1,85 @@
+## API Report File for "@azure/storage-blob-changefeed"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BlobServiceClient } from '@azure/storage-blob';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+
+// @public (undocumented)
```

Comment:
Can we add doc comments for these public types?

Reply:
Noted. Plan to add comments in next GA release. Is that fine with you?

Reply:
Sure, as long as it's tracked, there's no real rush. :)